aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -201,7 +201,7 @@ protected TableOperations newTableOps(TableIdentifier tableIdentifier) {
       Map<String, String> tableSpecificCatalogProperties = ImmutableMap.<String, String>builder()
           .putAll(catalogProperties)
           .put(AwsProperties.LAKE_FORMATION_DB_NAME,
-              IcebergToGlueConverter.getDatabaseName(tableIdentifier))
+              IcebergToGlueConverter.getDatabaseName(tableIdentifier, awsProperties.glueCatalogSkipNameValidation()))
           .put(AwsProperties.LAKE_FORMATION_TABLE_NAME,
```

Comment:
I think we have to also skip table name validation in the same way using the same config